tests/stub/driver_parameters/test_bookmark_manager.py

```diff
@@ -0,0 +1,598 @@
+import json
+import re
+
+from nutkit.frontend import (
+    DefaultBookmarkManagerConfig,
+    Driver,
+)
+import nutkit.protocol as types
+from tests.shared import TestkitTestCase
+from tests.stub.shared import StubServer
+
+
+class TestDefaultBookmarkManager(TestkitTestCase):
```

Comment:
nit: the default bookmark manager is a no-op, isn't it?

Reply:
Default implementation, not default configuration. But I will call it Neo4jBookmarkManager to not be confusing.